composer.json

```diff
@@ -62,7 +59,11 @@
     "drupal/webform": "^5.4",
     "drush/drush": "^10.1.0",
     "webflo/drupal-finder": "^1.1",
-    "zaporylie/composer-drupal-optimizations": "^1.0"
+    "zaporylie/composer-drupal-optimizations": "^1.0",
+    "drupal/core-composer-scaffold": "^8.8",
```

Comment:
Please don't add this manually, use `composer require` for that, it will keep packages sorted